src/main/resources/static/js/MainWindow.js

```diff
@@ -913,8 +913,8 @@ function importData() {
 										function() {
 											currentInput = $(this).find(
 													"td:eq(1) input").val();
-											if (currentInput === null
-													|| currentInput === "") {
+											if (currentInput == null
+													|| currentInput == "") {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)